api/net/Areas/Editor/Controllers/AVOverviewSectionItemController.cs

```diff
@@ -109,6 +113,20 @@ public IActionResult FindById(int id)
         return new JsonResult(new AVOverviewSectionItemModel(result, _serializerOptions));
     }
 
+    /// <summary>
+    /// Find AVOverviewSectionItem for the specified section 'id'.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+    [HttpGet("group/{id}")]
```

Comment:
Change `group/{id}` to `for/section/{sectionId}` so that the path is descriptive.  The path `group` isn't intuitive for the returned results.